### PR TITLE
Fix/namespaces

### PR DESCRIPTION
**Pull request by cicnavi**
This replaces namespaces so that they are PSR-4 compatible. 

Related to #65, however this solves only part of the problem. 

New error is related to Symfony DependencyInjection when 'usenewui' is set to true: 

```
SimpleSAML\Error\Error: UNHANDLEDEXCEPTION
Backtrace:
1 www/_include.php:17 (SimpleSAML_exception_handler)
0 [builtin] (N/A)
Caused by: Symfony\Component\DependencyInjection\Exception\RuntimeException: Cannot autowire service "SimpleSAML\Module\oidc\Controller\ClientCreateController": argument "$configurationService" of method "__construct()" references class "SimpleSAML\Module\oidc\Services\ConfigurationService" but no such service exists.
Backtrace:
10 vendor/symfony/dependency-injection/Compiler/DefinitionErrorExceptionPass.php:54 (Symfony\Component\DependencyInjection\Compiler\DefinitionErrorExceptionPass::processValue)
9 vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:81 (Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass::processValue)
8 vendor/symfony/dependency-injection/Compiler/DefinitionErrorExceptionPass.php:32 (Symfony\Component\DependencyInjection\Compiler\DefinitionErrorExceptionPass::processValue)
7 vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:46 (Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass::process)
6 vendor/symfony/dependency-injection/Compiler/Compiler.php:91 (Symfony\Component\DependencyInjection\Compiler\Compiler::compile)
5 vendor/symfony/dependency-injection/ContainerBuilder.php:749 (Symfony\Component\DependencyInjection\ContainerBuilder::compile)
4 vendor/symfony/http-kernel/Kernel.php:545 (Symfony\Component\HttpKernel\Kernel::initializeContainer)
3 vendor/symfony/http-kernel/Kernel.php:786 (Symfony\Component\HttpKernel\Kernel::preBoot)
2 vendor/symfony/http-kernel/Kernel.php:188 (Symfony\Component\HttpKernel\Kernel::handle)
1 lib/SimpleSAML/Module.php:199 (SimpleSAML\Module::process)
0 www/module.php:10 (N/A)
```

### CODE REVIEW

**Comment by tvdijen** [comment]
Isn't this error just a matter of missing routes.yml and services.yml files?

**Comment by cicnavi** [comment]
Thanks for the hint @tvdijen . 
I have added the services.yml and did some other smaller tweaks to get the module operational when the 'usenewui' is enabled.
Just to note here: we had a Controller directory / namespace defined from before "by accident", it was not the original intention to use it in a way that the Symfony Kernel loads classes from it when the new UI is enabled. 
I have defined services.yml (and empty routes.yml) just to have it as a TODO for SSP v2, but at this point the module works the same as before (it uses php files as routes, uses its own Contaner...). However, I think this module can greatly benefit from the Symfony DI.

File: docker/ssp/config-override.php [hunk 1]
```diff
@@ -6,5 +6,6 @@
         'database.username' => 'user',
         'database.password' => 'password',
         'language.i18n.backend' => 'gettext/gettext',
-        'logging.level' => 7
+        'logging.level' => 7,
+//        'usenewui' => true,
```
Comment (joukero) [inline docker/ssp/config-override.php:10]: We should probably make a choice here, either true, false or removed completely but not commented out.

File: lib/Services/RoutingService.php [hunk 2]
```diff
@@ -50,6 +50,7 @@ public static function call(string $controllerClassname, bool $authenticated = t
         $serverRequest = ServerRequestFactory::fromGlobals();
         $response = $controller($serverRequest);
 
+        # TODO sspv2 return Symfony\Component\HttpFoundation\Response (Template instance) in SSP v2
```
Comment (joukero) [inline lib/Services/RoutingService.php:53]: Is this something to fix before releasing these namespace changes?

File: docker/ssp/config-override.php (hunk 1 above)
Comment (cicnavi) [reply docker/ssp/config-override.php]: Ok, I've set it to false by default.

File: lib/Services/RoutingService.php (hunk 2 above)
Comment (cicnavi) [reply lib/Services/RoutingService.php:53]: No, this is just to mark a return type needed when we do an overhaul of the module to use Symfony DI, in SimpleSAMLphp v2. Now we are still using our own container and resolving instances ourselves.

**Pull request merged**